_config/site.yml

```
@@ -383,13 +383,15 @@ projects:
     name: Hibernate Tools
     icon: 'icon-wrench'
     description: |
-      Working with Hibernate is very easy and developers enjoy using the APIs and the query language.
-      Even creating mapping metadata is not an overly complex task once you've mastered the basics.
-      Hibernate Tools makes working with Hibernate or JPA even more pleasant.
-    blog_tag: JBoss Tools
-    blog_tag_url: jboss-tools
+      Working with Hibernate is very easy and developers enjoy using the APIs and the query language.<br/>
+      But sometimes you want to get started with a quick prototype or overcome the burden of creating lots of entities 
+      that reflect your database structure.<br/>
+      Hibernate Tools and its reverse engineering functionalities will help you and make 
+      working with Hibernate and JPA even more pleasant.
```

This is too long; it should be just a fairly short sentence.

```suggestion
      Reverse engineer Java entities from an existing SQL database schema.
```